website/content/docs/enterprise/license/faq.mdx

```diff
@@ -0,0 +1,91 @@
+Nomad Enterprise automatically load Nomad licenses when a Nomad server agent starts using Nomad Enterprise.
+
+## Q: Can I get a quick summary of the Nomad changes?
+
+Starting with Nomad Enterprise v1.1.0, the license enablement process is different.
+
+HashiCorp Enterprise servers will no longer start without a license. When the server instance initializes the license watcher, the server reads from either an environment variable or file. If the license is missing, invalid, or expired, the server will immediately exit. 
+This check is part of the server boot-up process.
+In previous versions of HashiCorp enterprise products, one server could distribute a license to other servers via the Raft protocol. 
+This will no longer work since each server must be able to find a valid license during the startup process.
+
+
+## Q: What resources are available?
+
+The list below is a great starting point for learning more about the license changes introduced in Nomad Enterprise v1.1.0
+
+- [v1.1.0 Upgrade Guide](https://www.nomadproject.io/docs/upgrade/upgrade-specific#enterprise-licenses)
+
+- [Nomad License Documentation](https://www.nomadproject.io/docs/enterprise/license)
+
+- [License configuration values documentation](https://www.nomadproject.io/docs/configuration/server#license_path)
+
+- [Install a HashiCorp Enterprise License Tutorial](https://learn.hashicorp.com/tutorials/nomad/hashicorp-enterprise-license)
+
+## Q: Does this affect Nomad or older versions of Nomad?
+
+This change only affects Nomad Enterprise version v1.1.0+ent and later.
+
+## Q: Does the license have to be present on Nomad client nodes?
+
+Only Nomad server nodes need to have the license present.
+Nomad agents running with the “-dev” flag will also need a license.
+
+## Q: What are some pitfalls that could be experienced during an upgrade because of the license changes?
+
+If you follow the steps in the [upgrade guide](https://www.nomadproject.io/docs/upgrade/upgrade-specific#enterprise-licenses), but have misconfigured the licensing steps, the server instances will stop during boot-up sequence. 
+
+If you do not check cluster health after adding new servers, and verify that the new servers have joined properly, then you may accidentally take down old servers without replacing them.
+
+If you are using a trial license in production, this could cause issues as the cluster will fail to start once the trial expires.
+
+If you have built automation around `nomad license put`, this command will fail as it has been removed from Nomad v1.1.0+.
+
+## Q: What are some other gotchas?
+
+You might not have access to your oragnization's Nomad license. 
+The  6-hour temporary license window is now removed, and a license is required for proper boot-up.
+Running `nomad agent -dev` without a license will fail.
+
+
+## Q: How can a user get their existing license to put on disk for an upgrade?
+
+Use `nomad license get` to retrieve the license, then save the license to disk.
```

Comment:
Unfortunately this isn't correct; the `nomad license get` returns the interpretation of the license (see https://www.nomadproject.io/docs/commands/license/get#examples) but not the license blob. A user will need to get their license from whereever they originally got it.

Reply:
@tgross I changed it to the following information.

## Q: How can a user get their existing license to put on disk for an upgrade?


If you are an existing HashiCorp enterprise customer you may contact your organization's customer success manager (CSM) or email support-softwaredelivery@hashicorp.com for information on how to get your organization's enterprise license.
You can use `nomad license get` to retrieve information about the license, but not the license itself.

Reply:
That works 👍